### DISCUSSION TITLE
"claimant and defendant" vs "claimant/applicant and defendant/respondent"

### DISCUSSION THREAD

**Discussion by yjyao** [passamo9/LaTeX_EnglandWales_Templates#23] [Q&A]
in the [Particulars of Claim example doc](https://github.com/passamo9/LaTeX_EnglandWales_Templates/blob/main/Particulars%20of%20Claim%2C%20first%20page.docx) it says "between _claimant_ and _defendant_," while the [Skeleton Arg example doc](https://github.com/passamo9/LaTeX_EnglandWales_Templates/blob/main/Skeleton%20Argument%20Template%2C%20first%20pages.docx) says "between _claimant/applicant_ and _defendant/respondent_." i'd like to figure out when to choose what version.

1. are both the Particulars of Claim and the Skeleton Arg types of pleadings?
2. does a "Particulars of Claim" doc always say "claimant and defendant," and a Skeleton Arg always say "claimant/applicant and defendant/respondent"?
3. are there other expressions used in place of "claimant and defendant"?

Ping @passamo9 for answers.

another way to look at these questions is, i'm trying to figure out a way to compose the titles that makes the most sense. for example, if 2 is true, then we can have

```tex
\documentclass[particularsofclaim]{engwales-legal}
...
\maketitle
```

so that it automatically says "claimant and defendant" since the user specified `particularsofclaim`. if the use of "claimant" vs "applicant" vs other possible expressions are not tied to the type of the pleading, and one often has to choose the most suitable word when writing a Particulars of Claim / other doc, then it would make more sense to write

```tex
\documentclass[particularsofclaim]{engwales-legal}
\aparty{claimant}
\bparty{defendant}
...
\maketitle
```

**Comment by passamo9** [comment]
1. Yes, they are both types of pleading. Generally, the Particulars of Claim are used to start a claim, and a Skeleton is prepared for a judge about a week before the hearing, so they are at used at the opposite ends of litigation—at the start and end.

2. Particulars of Claim has (a) Claimant(s), (b) Defendant(s), and additionally (rarely) (c) Third Parties, or (d) Intervenors. I wanted to keep it simple for now, then add (c) and (d) later. Maybe these could be added with a `%` comment in case either is needed.

3. Skeleton arguments, used in interim hearings, include references to who is making an application for a hearing. Not always is the claimant the applicant. In some cases, a Defendant may make an application to strike out part of a claim, etc. In which case, it would be Claimant/Respondent and Defendant/Applicant. There are also situations in which the Respondent/Applicant is changed to Appellant/Respondent in case of appeals. But generally, skeleton arguments do have this double-barrelled feature.

4. Claimant and Defendant are stable terms. Unlike in the use, "Plaintiff" is never used instead of Claimant.

Thank you so much for your explanation! Super clear and I appreciate the detail and thought you are putting into this!
